Components/Queues/Drivers/SyncQueueDriverInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers;
+
+
+interface SyncQueueDriverInterface extends QueueDriverInterface
+{
+
+    /**
+     * Returns true if a subscription already exists, false otherwise.
+     *
+     * @return bool
+     */
+    public function isSubscribed();
+
+    /**
+     * Creates a subscription to the given $queue, allowing to receive messages from it.
+     *
+     * @param string $queue Queue to subscribe
+     */
+    public function subscribe($queue);
```

Comment:
We could add the type hint **string** for the $queue variable

Reply:
Done.

Reply:
I see you added this, but the interface doesn't have `declare(strict_types=1)`